core/src/main/resources/org/apache/spark/ui/static/jquery.cookies.2.2.0.min.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2005 - 2010, James Auldridge
```

Comment:
Hi @tgravescs , addressed all your comments (removing functions and page index, add licenses and showInComplete). 
I tried deleting cookies, that will cause reference errors since it is used by datatable, etc.